package.json

```diff
@@ -46,18 +46,18 @@
     "stylelint-scss": "5.0.1",
     "stylelint-use-nesting": "4.1.0",
     "tslib": "^2.2.0",
-    "zone.js": "~0.13.3"
+    "zone.js": "~0.14.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^16.2.12",
-    "@angular-eslint/builder": "16.3.1",
-    "@angular-eslint/eslint-plugin": "16.3.1",
-    "@angular-eslint/eslint-plugin-template": "16.3.1",
-    "@angular-eslint/schematics": "16.3.1",
-    "@angular-eslint/template-parser": "16.3.1",
-    "@angular/cli": "^16.2.12",
-    "@angular/compiler-cli": "^16.2.12",
-    "@angular/material-date-fns-adapter": "16.2.14",
+    "@angular-devkit/build-angular": "^17.3.6",
+    "@angular-eslint/builder": "17.3.0",
+    "@angular-eslint/eslint-plugin": "17.3.0",
+    "@angular-eslint/eslint-plugin-template": "17.3.0",
+    "@angular-eslint/schematics": "17.3.0",
+    "@angular-eslint/template-parser": "17.3.0",
+    "@angular/cli": "^17.3.6",
+    "@angular/compiler-cli": "^17.3.6",
+    "@angular/material-date-fns-adapter": "17.3.6",
     "@ckeditor/ckeditor5-angular": "^5.2.0",
     "@ckeditor/ckeditor5-build-classic": "^37.0.0",
     "@enigmatry/scss-foundation": "15.0.0-preview.5",
```

Comment:
we're at 16.1 already, shouldn't we upgrade this 1?

Reply:
This feels like it shouldn't even be here, it means we are depending on ourselves?

Reply:
Not sure, I think we use these to test if theme and components styles compile (they reference scss-foundation)
I would leave it for now.